aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -366,16 +391,29 @@ module aptos_framework::fungible_asset {
             balance: 0,
             frozen: false,
         });
+
+        if (default_to_concurrent_fungible_balance()) {
+            move_to(store_obj, ConcurrentFungibleBalance {
+                balance: aggregator_v2::create_unbounded_aggregator(),
```

Comment:
Do we want unbounded here? Should `create_store` be aware of the limit?

Reply:
it might be that limit can change. then it would be a nightmare if balances have different limits stored.

they are not minting/burning here, so checking against the limit is pointless. this method should never be able to overflow.

Reply:
it's like u64... it has an implicit bound of U64::MAX, so does unbound aggregator.

Reply:
yes, but errors are different. For unbounded, it will be a nice move error, for u64::max it will be actually a runtime internal error on overflow. It is fine this way, but for consistency would be nice to have similar APIs and error types

Reply:
code seems fine, "unbounded" only please, otherwise we get quirky behavior that doesn't exist in the basic FAS

also a better conversation for the AIP